lib/school_house_web/templates/page/index.html.heex

```diff
@@ -10,25 +10,26 @@
       </p>
       <p class="mb-6 leading-relaxed prose dark:prose-dark">
         <%= gettext("Through the hard work of volunteers Elixir School has been translated to many languages. Some of these translations include: ")%>
-        <a href="/vi" title="Vietnamese">Việt ngữ</a>,
-        <a href="/zh-hans" title="Chinese, in simplified script">简体中文</a>, 
-        <a href="/zh-hant" title="Chinese, in traditional script">繁體中文</a>, 
+        <a href="/bn" title="Bengali">বাংলা</a>,
+        <a href="/de" title="German">Deutsch</a>,
+        <a href="/el" title="Modern Greek">Ελληνικά</a>.
         <a href="/es" title="Spanish">Español</a>, 
-        <a href="/sk" title="Slovak">Slovenčina</a>, 
+        <a href="/id" title="Indonesian">Bahasa Indonesia</a>,
+        <a href="/it" title="Italian">Italiano</a>,
         <a href="/ja" title="Japanese">日本語</a>,
+        <a href="/ko" title="Korean">한국어</a>,
+        <a href="/ms" title="Malay">Bahasa Melayu</a>,
+        <a href="/no" title="Norwegian">Norsk</a>,
         <a href="/pl" title="Polish">Polski</a>,
         <a href="/pt" title="Portuguese">Português</a>,
+        <a href="/vi" title="Vietnamese">Việt ngữ</a>,
+        <a href="/zh-hans" title="Chinese, in simplified script">简体中文</a>, 
+        <a href="/zh-hant" title="Chinese, in traditional script">繁體中文</a>, 
         <a href="/ru" title="Russian">Русский</a>,
-        <a href="/id" title="Indonesian">Bahasa Indonesia</a>,
-        <a href="/ms" title="Malay">Bahasa Melayu</a>,
-        <a href="/uk" title="Ukrainian">Українською</a>,
-        <a href="/ko" title="Korean">한국어</a>,
-        <a href="/it" title="Italian">Italiano</a>,
-        <a href="/de" title="German">Deutsch</a>,
-        <a href="/bn" title="Bengali">বাংলা</a>,
+        <a href="/sk" title="Slovak">Slovenčina</a>, 
         <a href="/tr" title="Turkish">Türkçe</a>,
         <a href="/th" title="Thai">ภาษาไทย</a>, <%= gettext("and")%>
-        <a href="/el" title="Modern Greek">Ελληνικά</a>.
+        <a href="/uk" title="Ukrainian">Українською</a>,
```

Comment:
```suggestion
        <a href="/uk" title="Ukrainian">Українською</a>.
```

This change will likely be unnecessary once the list is alphabetized, and it's an un-important nitpick, but if you don't mind ending the list with a period that would be great 🙂

Reply:
Ah! That is a very fair point! 😄 

It should be good now 👍